Makefile

```diff
@@ -67,7 +67,10 @@ clean: ## Clean all the temporary files
 	@rm -rf ./bin
 	@rm -rf ./test/output
 
-test: test-alipay test-wechat ## Run all tests
+test: test-go test-alipay test-wechat ## Run all tests
+
+test-go: ## Run Golang tests
+	@go test ./...
 
 test-alipay: ## Run tests for Alipay provider
 	@$(SHELL) ./test/alipay-test.sh
```

Comment:
Can we add `[PASS]` assertion in these test scripts?

If the stdout doesn't have the `[PASS]`, then exit with 1 (error).

Maybe like this:
```bash
./test/alipay-test.sh | grep "[PASS]" > /dev/null || exit 1
```

Other providers are the same.

Reply:
never mind.

Reply:
I am not sure why we need it. The process will return 1 if there are test failures.